packages/utils/src/lib/mock/schema-helper.mock.ts

```diff
@@ -78,11 +78,11 @@ export function mockAuditConfig(opt?: {
 }
 
 export function mockPersistConfig(opt?: Partial<PersistConfig>): PersistConfig {
-  let { outputPath, format } = opt || {};
-  outputPath = outputPath || __outputPath__;
+  let { outputDir, format } = opt || {};
+  outputDir = outputDir || __outputFile__;
```

Comment:
This looks like a mistake. Should be probably be `'tmp'`.

Reply:
done

Reply:
Is it? I still see the same diff :confused: 

![image](https://github.com/flowup/quality-metrics-cli/assets/34691111/e1498015-d974-486f-83a4-750578347fc1)

Reply:
now